Modules/rat_rescue.py

```diff
@@ -121,54 +122,15 @@ def __eq__(self, other) -> bool:
             # instance type check
             return NotImplemented
         else:
-            # check equality
-
-            conditions = [
-                self.uuid == other.uuid,
-                self.board_index == other.board_index,
-                self.client == other.client,
-                self.rats == other.rats,
-                self.platform == other.platform,
-                self.first_limpet == other.first_limpet,
-                self.created_at == other.created_at,
-                self.updated_at == other.updated_at,
-                self.system == other.system,
-                self.unidentified_rats == other.unidentified_rats,
-                self.active == other.active,
-                self.code_red == other.code_red,
-                self.outcome == other.outcome,
-                self.title == other.title,
-                self.first_limpet == other.first_limpet,
-                self.marked_for_deletion == other.marked_for_deletion,
-                self.lang_id == other.lang_id,
-                self.rats == other.rats,
-                self.irc_nickname == other.irc_nickname,
-            ]
-
-            return all(conditions)
+            # abide by python equality contract.
+            return hash(self) == hash(other)
 
     def __hash__(self):
 
         if self._hash is None:
-            attributes = (
-                self.uuid,
-                self.board_index,
-                self.client,
-                self.platform,
-                self.first_limpet,
-                self.created_at,
-                self.updated_at,
-                self.system,
-                self.active,
-                self.code_red,
-                self.outcome,
-                self.title,
-                self.first_limpet,
-                self.lang_id,
-                self.irc_nickname,
-            )
-
-            self._hash = reduce(xor, map(hash, attributes))
+            # compute the hash of the internally generated uuid and use that as our hash
+            self._hash = hash(self.__hashing_uuid)
+
         return self._hash
```

Comment:
This doesn't change much, but I think we can drop the lazy initialization now that the algorithm is so much simpler and the hashing id doesn't change.